pkg/kubectl/rolling_updater.go

```diff
@@ -61,24 +61,26 @@ type RollingUpdaterConfig struct {
 	// complete.
 	CleanupPolicy RollingUpdaterCleanupPolicy
 	// The maximum number of pods that can be unavailable during the update.
-	// Value can be an absolute number (ex: 5) or a percentage of total pods at
-	// the start of update (ex: 10%). Absolute number is calculated from
-	// percentage by rounding up. This can not be 0 if MaxSurge is 0. By
-	// default, a fixed value of 1 is used. Example: when this is set to 30%,
-	// the old RC can be scaled down by 30% immediately when the rolling update
-	// starts. Once new pods are ready, old RC can be scaled down further,
-	// followed by scaling up the new RC, ensuring that at least 70% of original
-	// number of pods are available at all times during the update.
+	// Value can be an absolute number (ex: 5) or a percentage of desired pods (ex: 10%).
```

Comment:
Minor: while you're at it, can you update these to use godoc conventions (starting the sentence with the type name)? e.g. "MaxUnavailable is the maximum number of pods [...]"